zppy/templates/mpas_analysis.bash

```diff
@@ -139,6 +139,28 @@ parallelTaskCount = {{ parallelTaskCount }}
 # handle 12 simultaneous processes, one for each monthly climatology.
 ncclimoParallelMode = bck
 
+# the number of total threads to use when ncclimo runs in "bck" or "mpi" mode.
+# Reduce this number if ncclimo is crashing (maybe because it is out of memory).
+# The number of threads must be a factor of 12 (1, 2, 3, 4, 6 or 12).
+ncclimoThreads = 12
+
```

Comment:
This is the MPAS-Analysis default but we might want to make this a zppy config option, too, for big meshes.